test/lib/clients/inst_productsources_test.rb

```diff
@@ -46,6 +49,16 @@
 
       client.main
     end
+
+    it "sets the RELEASEVER_ENV to Product.version" do
+      stub_const("#{described_class}::RELEASEVER_ENV", "ZYPP_RELEASEVER_ENV")
```

Comment:
is this needed? why stubbing it?

Reply:
Hmmm, probably not. Re-reading my own test now, I realize that the important part is that the code set the "ZYPP_RELEASEVER_ENV" no matter how we called/stored it internally :)

Thanks!